containers/withQueryLoader.tsx

```diff
@@ -23,8 +24,13 @@ const withQueryLoader = <T extends {}>(
   if (data) {
     return <Component queryData={data} {...props} />
   }
-
-  return <h1>No Data...</h1>
+  return (
+    <Error
+      title="Internal server error"
+      message="No data"
+      src="/500.png"
+    ></Error>
```

Comment:
`</Error>` should not be needed, you can just use a self enclosing tag since the component does not display children. 

It is in our engineering practices:
https://github.com/garageScript/c0d3-app/wiki/Engineering-Practices